pkg/scheduler/reconciliation/persistent.go

```diff
@@ -95,8 +95,14 @@ func (r *PersistentReconciliationRepository) CreateReconciliation(state *cluster
 		for idx, components := range reconSeq.Queue {
 			priority := idx + 1
 			for _, component := range components {
+
+				componentPriority := priority
+				if component.Component == "service-manager-proxy" {
```

Comment:
Sorry, hard-coded component-names is a no-go. 

Components should not have dependencies and installation-order is only allowed for a very small set of components (like istio, cluster-essentials).

Why has the service-manager-proxy to be installed at the end of the installation?

=> Please see https://github.com/kyma-incubator/reconciler/issues/567 - this is about dealing with enhanced ordering.

But in general: a Kyma component (except if if's a very special mandatory component) should not care when it get's installed. We are retrying installations and a component should rely on the retry.